### PR TITLE
Uniquify ROOT file name when output is directed to stdout

### PR DESCRIPTION
**Pull request by pcanal**
Use `stdout-${PID}.root` for the ROOT outputname.

### CODE REVIEW

File: app/celer-g4/RootIO.cc [hunk 1]
```diff
@@ -54,6 +54,11 @@ RootIO::RootIO()
         file_name_ = "celer-g4.root";
     }
 
+    if (file_name_ == "-")
+    {
+        file_name_ = "stdout-" + std::to_string(::getpid()) + ".root";
```
Comment (sethrj) [inline app/celer-g4/RootIO.cc:63]: I'm sure ROOT leaks the include, but this requires `#include <unistd.h>` on POSIX machines... does it require anything on windows?

**Review by sethrj — CHANGES_REQUESTED** [review]

File: app/celer-g4/RootIO.cc (hunk 1 above)
Comment (pcanal) [reply app/celer-g4/RootIO.cc:63]: It does need `process.h`.  What is our Windows marker/ifdef?

Comment (pcanal) [reply app/celer-g4/RootIO.cc:63]: It is the (misnamed) `_WIN32`.

File: app/celer-g4/RootIO.cc [hunk 2]
```diff
@@ -28,6 +28,10 @@
 #include "GlobalSetup.hh"
 #include "SensitiveHit.hh"
 
+#ifdef _WIN32
+#    include <process.h>
```
Comment (sethrj) [inline app/celer-g4/RootIO.cc:32]: Should this be
```suggestion
#ifndef _WIN32
#    include <unistd.h>
#else
#    include <process.h>
```
maybe? (or `defined(__APPLE__) || defined(__linux__)`?)

Comment (sethrj) [reply app/celer-g4/RootIO.cc:32]: We can IWYU later.

**Review by sethrj — APPROVED** [review]

File: app/celer-g4/RootIO.cc (hunk 2 above)
Comment (pcanal) [reply app/celer-g4/RootIO.cc:32]: I prefer
```
#ifdef __WIN32
do something silly
#else
do something sensitive
#endif
```
(or `#ifndef `)

Comment (pcanal) [reply app/celer-g4/RootIO.cc:32]: You merged while the commit was moving towards github :)

**Pull request merged**